src/SprykerEco/Yves/Easycredit/Plugin/Router/EasycreditRouteProviderPlugin.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\Easycredit\Plugin\Router;
+
+use Spryker\Yves\Router\Plugin\RouteProvider\AbstractRouteProviderPlugin;
+use Spryker\Yves\Router\Route\RouteCollection;
+use Symfony\Component\HttpFoundation\Request;
+
+class EasycreditRouteProviderPlugin extends AbstractRouteProviderPlugin
+{
+    /**
+     * @var string
+     */
+    public const ROUTE_NAME_EASYCREDIT_SUCCESS_RESPONSE = 'easycredit/success-response';
+
+    /**
+     * {@inheritDoc}
+     * - Adds Easycredit success response route to the RouteCollection.
+     *
+     * @api
+     *
+     * @param \Spryker\Yves\Router\Route\RouteCollection $routeCollection
+     *
+     * @return \Spryker\Yves\Router\Route\RouteCollection
+     */
+    public function addRoutes(RouteCollection $routeCollection): RouteCollection
+    {
+        $route = $this->buildRoute('/easycredit/payment/success', 'Easycredit', 'Easycredit', 'successEasycreditResponse');
+        $route = $route->setMethods([Request::METHOD_GET]);
+        $routeCollection->add(static::ROUTE_NAME_EASYCREDIT_SUCCESS_RESPONSE, $route);
```

Comment:
Would it make sense to extract this to a separate method for a future extension?
E.g. `addPaymentSuccessRoute`

Reply:
I'm not sure. I guess it could be done in the future if such case occurs and for now to leave it like this in favor of YAGNI. What do you think?